### DISCUSSION TITLE
Why isn't alignSelf exposed on <Flex>?

### DISCUSSION THREAD

**Discussion by samselikoff** [radix-ui/themes#520] [Q&A]
Hi there! Just curious about why certain properties like alignSelf aren't exposed on <Flex>. Usually there's another way to achieve the desired result, but I'm more curious about the rationale behind not exposing it so I can be aligned with the library's design decisions.

Thanks for this excellent lib!

**Comment by vladmoroz** [comment]
For `alignSelf` and `justifySelf` specifically, we generally try to avoid making it ergonomic for a component to override the parent’s layout. Saw it get out of hand, and as you said, there usually are workarounds.

**Reply by vladmoroz** [reply]
Although, as I was writing it, I realised that properties like `gridColumnStart` effectively achieve the same, so there’s a precedent

**Reply by samselikoff** [reply]
That makes sense!
